app/views/foreman_ansible/job_templates/ansible_-_inventory.erb

```diff
@@ -0,0 +1,30 @@
+<%#
+name: Ansible Inventory
+snippet: false
+template_inputs:
+- name: required
+  required: false
+  input_type: user
+  description: Comma-separated values that should be returned.
```

Comment:
I think it would be nicer user experience if we define separate input for every optional part, it would have restricted values yes no and would be required. That in UI should generate select boxes to chose from yes/no.

We could also use this chance to add default values to inputs (that would be PR to Foreman core)

Reply:
Done: https://github.com/theforeman/foreman/pull/7050

Reply:
Code there looks good, but the use-case seems bit odd to me. @ares, how another editable field is going to stop me from making a mess, if I am a messy user? Can't we have lockable report_templates instead?

Reply:
if not, `Ansible Inventory` is suggestive enough for me to get an idea what that might be used for and why I shall not mess with it IMHO.